packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/Providers/ClientProviders/TestData/ClientProviderTests/ValidateClientWithSpread.cs

```diff
@@ -0,0 +1,82 @@
+// <auto-generated/>
+
+#nullable disable
```

Comment:
lets trim down what text we are validating here.  We might even be able to do this all in memory since we are just validating the signature of a method?

Reply:
I've removed this and do the validation in memory.